Assets/Scripts/ProjectileHazardScript.cs

```diff
@@ -1,12 +1,13 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 
 public class ProjectileHazardScript : MonoBehaviour
 {
-    [SerializeField] float strengthOfAttraction = 5f;
-    [SerializeField] float pullRadius = 10f;
-    [SerializeField] GameObject playerRocket;
+    private float strengthOfAttraction = 10f;
+    private float pullRadius = 12f;
```

Comment:
Similarly this isn't really the pull radius. This should be called something like `detectionRadius`